examples/material_gallery/lib/gallery/home.dart

```diff
@@ -49,109 +49,81 @@ class GalleryHome extends StatefulWidget {
 }
 
 class GalleryHomeState extends State<GalleryHome> {
+  Key _scrollableKey = new UniqueKey();
+
   @override
   Widget build(BuildContext context) {
-    final double appBarHeight = 128.0;
     return new Scaffold(
       drawer: new GalleryDrawer(),
       appBar: new AppBar(
-        expandedHeight: appBarHeight,
-        flexibleSpace: (BuildContext context) {
-          return new Container(
-            padding: const EdgeInsets.only(left: 64.0),
-            height: appBarHeight,
-            child: new Align(
-              alignment: const FractionalOffset(0.0, 1.0),
-              child: new Text('Flutter Gallery', style: Typography.white.headline)
-            )
-          );
-        }
+        expandedHeight: 256.0,
```

Comment:
This should be an appBarHeight constant, since it appears in the padding as well.